haystack/document_stores/elasticsearch/es7.py

```diff
@@ -7,7 +7,7 @@
 from haystack.schema import Document, FilterType
 from haystack.document_stores.filter_utils import LogicalFilterClause
 from haystack.lazy_imports import LazyImport
-from .search_engine import SearchEngineDocumentStore, prepare_hosts
+from ..search_engine import SearchEngineDocumentStore, prepare_hosts
```

Comment:
Why keep it relative? This should work fine too right?

```suggestion
from haystack.document_stores.search_engine import SearchEngineDocumentStore, prepare_hosts
```